src/extensions/registries/page-menu-registry.ts

```diff
@@ -43,33 +37,23 @@ export class GlobalPageMenuRegistry extends BaseRegistry<PageMenuRegistration> {
   }
 }
 
-export class ClusterPageMenuRegistry extends BaseRegistry<ClusterPageMenuRegistration> {
-  @action
-  add(items: PageMenuRegistration[], ext: LensExtension) {
-    const normalizedItems = items.map(menuItem => {
-      menuItem.target = {
-        extensionId: ext.name,
-        ...(menuItem.target || {}),
-      };
-
-      return menuItem;
-    });
-
-    return super.add(normalizedItems);
-  }
-
+export class ClusterPageMenuRegistry extends PageMenuRegistry<ClusterPageMenuRegistration> {
   getRootItems() {
     return this.getItems().filter((item) => !item.parentId);
   }
 
   getSubItems(parent: ClusterPageMenuRegistration) {
-    return this.getItems().filter((item) => item.parentId === parent.id && item.target.extensionId === parent.target.extensionId);
+    return this.getItems().filter((item) => {
+      return item.parentId === parent.id && item.target.extensionId === parent.target.extensionId;
+    });
   }
 
-  getByPage(page: RegisteredPage) {
-    return this.getItems().find((item) => item.target?.pageId == page.id && item.target?.extensionId === page.extensionId);
+  getByPage({ id: pageId, extensionId }: RegisteredPage) {
+    return this.getItems().find((item) => {
+      return item.target.pageId == pageId && item.target.extensionId === extensionId;
```

Comment:
I don't think we need the return here either.

Reply:
Are we still talking about code-formatting? o_O

Reply:
Code-formatting is valid to discuss in PRs (even if not the most important thing), but I'm mostly commenting since the formatting was changed for no apparent reason and I don't think it's an improvement.

But I do agree that eslint should handle these. Until we have more full eslint rules it's up to discussion in my opinion.